PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/views/ServiceExplorerView.java

```diff
@@ -240,8 +240,12 @@ public void listChanged(final ListChangedEvent e) {
             switch (e.getAction()) {
             case add:
                 // create child tree nodes for the new nodes
-                for(Node childNode : (Collection<Node>)e.getNewItems()) {
-                    treeNode.add(createTreeNode(childNode));
+                for (Node childNode : (Collection<Node>) e.getNewItems()) {
+                    //Dirty fix here, for issue https://github.com/Microsoft/azure-tools-for-java/issues/2791
+                    //Since we do not support slot, here should not let user see slot in the azure explorer
+                    if (!childNode.getName().toLowerCase().contains("slot")) {
```

Comment:
this may stop rendering all the other resources containing "slot" in name.

Reply:
Use equals instead of contains